api/HTMLElement.json

```diff
@@ -1969,10 +1969,10 @@
               "version_added": "45"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
Where did the "10" come from. WebKit source has `nonce` in some IDL files, so this is a more complicated story that needs more digging.